tests/unit/db/db_updates_tests.py

```diff
@@ -0,0 +1,469 @@
+#!/usr/bin/env python
+# Copyright (c) 2016 IBM. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for _db_updates feed
+"""
+
+import unittest
+from requests import Session
+import json
+import os
+
+from cloudant.feed import Feed
+from cloudant.document import Document
+from cloudant.design_document import DesignDocument
+from cloudant.error import CloudantArgumentError, CloudantException
+from cloudant._2to3 import unicode_
+
+from .unit_t_db_base import UnitTestDbBase
+from ... import BYTETYPE
+
+class DbUpdatesTestsBase(UnitTestDbBase):
+    """
+    Common _db_updates tests methods
+    """
+
+    def setUp(self):
+        """
+        Set up test attributes
+        """
+        super(DbUpdatesTestsBase, self).setUp()
+        self.client.connect()
+        self.new_dbs = list()
+
+    def tearDown(self):
+        """
+        Reset test attributes
+        """
+        [db.delete() for db in self.new_dbs]
+        self.client.disconnect()
+        super(DbUpdatesTestsBase, self).tearDown()
+
+    def create_dbs(self, count=3):
+        self.new_dbs += [(self.client.create_database(self.dbname())) for x in range(count)]
+
+@unittest.skipIf(os.environ.get('RUN_CLOUDANT_TESTS'),
+    'Skipping CouchDB _db_updates feed tests')
+class CouchDbUpdatesTests(DbUpdatesTestsBase):
+    """
+    CouchDB _db_updates feed unit tests
+    """
+    def test_constructor_db_updates(self):
+        """
+        Test constructing a _db_updates feed
+        """
+        feed = Feed(self.client, feed='continuous', heartbeat=False, timeout=2)
+        self.assertEqual(feed._url,
+            '/'.join([self.client.cloudant_url, '_db_updates']))
+        self.assertIsInstance(feed._r_session, Session)
+        self.assertFalse(feed._raw_data)
+        self.assertDictEqual(feed._options,
+            {'feed': 'continuous', 'heartbeat': False, 'timeout': 2})
+
+    def test_stop_iteration_of_continuous_feed_with_heartbeat(self):
+        """
+        - Test stopping the iteration
+        - Test continuous feed
+        - Test heartbeat
```

Comment:
Probably better to keep the comments consistent by using `and` instead of bullet points.